src/main/types/SIOP.types.ts

```diff
@@ -637,9 +637,9 @@ export const isRequestPayload = (object: AuthenticationRequestPayload | Authenti
 export const isResponsePayload = (object: AuthenticationRequestPayload | AuthenticationResponsePayload): object is AuthenticationResponsePayload =>
   'iss' in object && 'aud' in object;
 
-export const isInternalVerification = (object: InternalVerification | ExternalVerification): object is InternalVerification =>
+export const isInternalVerification = (object: Verification): object is InternalVerification =>
   object.mode === VerificationMode.INTERNAL; /* && !isExternalVerification(object)*/
-export const isExternalVerification = (object: InternalVerification | ExternalVerification): object is ExternalVerification =>
+export const isExternalVerification = (object: Verification): object is ExternalVerification =>
```

Comment:
Don't change this to the super interface. Right now this interface is exactly the same as the InternalVerification. But if we add one unique prop to InternalVerification you would still be able to pass in objects with to few params, as long as they have all props from Verification. It really needs to be either Internal or External Verification

Reply:
reverted to `InternalVerification | ExternalVerification`